maps/migrations/0004_auto_20150201_1418.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('maps', '0003_auto_20141123_1852'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='map',
+            name='has_vulnerable_population_data',
+            field=models.BooleanField(default=False, help_text=b'Does the map show information on specific vulnerabilities of the population.'),
+            preserve_default=True,
+        ),
+        migrations.AddField(
+            model_name='map',
+            name='vulnerable_population_data_date_earliest',
+            field=models.DateField(null=True, blank=True),
+            preserve_default=True,
+        ),
+        migrations.AddField(
+            model_name='map',
+            name='vulnerable_population_data_date_latest',
+            field=models.DateField(null=True, blank=True),
+            preserve_default=True,
+        ),
+        migrations.AddField(
+            model_name='map',
+            name='vulnerable_population_data_source',
+            field=models.ManyToManyField(related_name='vulnerability_data_source_for', null=True, to='maps.DataSource', blank=True),
+            preserve_default=True,
+        ),
+        migrations.AlterField(
```

Comment:
hm I wonder why it thinks this has changed... I must have changed something before. Oops.